src/rred_reports/reports/interface.py

```diff
@@ -41,7 +41,7 @@ def validate_data_sources(year: int, template_file: Path, top_level_dir: Optiona
         dict: Dictionary of validated data sources
     """
     if top_level_dir is None:
-        top_level_dir = Path(__file__).resolve().parents[3]
+        top_level_dir = Path(__file__).resolve().parents[6]
```

Comment:
this seems surprising that it changed

Reply:
actually meant to flag this (there is another one of these L155 and L247 of `reports/emails.py`) - this was necessary when running via the CLI as `__file__` was resolving as a library directory inside the virtual environment rather than the relative file path from the top level of the repo.